indico/modules/events/editing/client/js/models.js

```diff
@@ -25,3 +25,9 @@ export const EditingReviewAction = {
   update: 'update',
   requestUpdate: 'request_update',
 };
+
+export const EditingType = {
+  paper: 'Paper',
+  slides: 'Slides',
+  poster: 'Poster',
+};
```

Comment:
My initial suggestion about replicating the enum was for cases where you need to check the internal name (ie the lowercased version). If you want a mapping for the user-facing name fine, but then it should be named e.g. `EditableTypeTitles` and use `Translate.string()`